design-tokens/assets/android/StyleDictionaryWithReferences.kt

```diff
@@ -0,0 +1,19 @@
+
+
+// Do not edit directly
+// Generated on Fri, 20 Jan 2023 15:09:34 GMT
+
+
+
+package io.element.compound.tokens;
+
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.*
+
+object StyleDictionary {
+  /** The default font size */
+  val sizeDefault = 256.00.dp
+  /** The main brand color */
+  val brand = #0dbd8b
+}
```

Comment:
I see no diff with the file `StyleDictionnary.kt`. What is the purpose of this `object` (it is maybe not necessary)?